javascript-light-bulb/javascript-light-bulb/main.js

```diff
@@ -0,0 +1,17 @@
+const button = document.querySelector('button');
+const html = document.querySelector('html');
```

Comment:
I'd recommend to select the `body` instead of the `html` tag for this, generally all visible parts of the page are going to be contained in the body so css rules should only be from the `body` tag and all it's children.

Reply:
updated the three files for `body` 
![Capture](https://github.com/aehrenkranz/m1-0723-code-solutions/assets/130402253/71165344-f960-43f1-ac35-89a97c86c2de)
element instead